src/components/BeaconResults.vue

```diff
@@ -294,6 +294,7 @@ export default {
       vm.filteringTerms = [];
       if (process.env.VUE_APP_DEVELOPMENT) {
         var wss = vm.aggregator.replace("http", "ws"); // change aggregator http url to ws
```

Comment:
Just as a side note, though not really a part of the review, this line should cover the line 299 as well rendering the check for dev environment redundant. General replace of `http` with `ws` changes `https` -> `wss` as well, since it doesn't touch the trailing `s`.

Reply:
The dev env check is needed for the next line `wss = wss.replace("0.0.0.0", "localhost");` to work. Which is needed so we can pass cookies locally to the backend. The .replace could be changed as you suggested but since the dev env check can't be removed i think the .replace should be left as is for clarity.